lib/app/components/pages/home/views/home_view.dart

```diff
@@ -70,37 +84,39 @@ class HomeView extends GetView<HomeController> {
                           },
                           separatorBuilder: (context, index) =>
                               const SizedBox(height: 10),
-                        )
-                      : Column(
-                          children: const [
-                            SizedBox(height: 30),
-                            Center(
-                              child: Text(
-                                '検索結果がありません',
-                                style: TextStyle(fontSize: 18),
-                              ),
+                        ))
+                    : Column(
+                        children: const [
+                          SizedBox(height: 30),
+                          Center(
+                            child: Text(
+                              '検索結果がありません',
+                              style: TextStyle(fontSize: 18),
                             ),
-                          ],
-                        ),
-                ),
+                          ),
+                        ],
+                      ),
               ),
             ),
           ),
-          Center(
-            child: ElevatedButton(
-              style: ElevatedButton.styleFrom(
-                fixedSize: Size(MediaQuery.of(context).size.width * 0.3, 25),
-                backgroundColor: const Color.fromRGBO(236, 188, 179, 1),
-                foregroundColor: Colors.white,
-                shape: const StadiumBorder(),
-                elevation: 0,
-              ),
-              onPressed: () {
-                // todo トップに戻る
-              },
-              child: const Text('TOPに戻る'),
-            ),
-          )
+          Obx(() => Visibility(
+              visible: controller.needScrollToTop.value,
+              child: Center(
```

Comment:
```suggestion
              child: Position(
                bottom: 25,
                left: Get.width / 3,
```
真ん中配置だと目立って落ち着かないので一旦の方に配置しましょう。
（上だとユーザが片手で持ってる時に指を伸ばさないといけないのでタップしにくいと思ってとりあえず下配置にしたい）
デザインに反映できてないですが、
イメージとしてはスクショのようにしたいです。
<img src="https://user-images.githubusercontent.com/25321380/204144107-bdcd8aec-12a5-43fc-bcdc-6cb6a6295037.png" width=250>

Reply:
ご指摘ありがとうございます。修正いたしました。